example_test.go

```diff
@@ -138,22 +138,9 @@ func ExampleSession_concurrency() {
 func ExampleDial_usingSSL() {
 	// To connect via TLS/SSL (enforced for MongoDB Atlas for example) requires
 	// configuring the dialer to use a TLS connection:
-	url := "mongodb://localhost:40003"
+	url := "mongodb://localhost:40003?ssl=true"
 
-	tlsConfig := &tls.Config{
-		// This can be configured to use a private root CA - see the Credential
-		// x509 Authentication example.
-		//
-		// Please don't set InsecureSkipVerify to true - it makes using TLS
-		// pointless and is never the right answer!
-	}
-
-	dialInfo, err := ParseURL(url)
-	dialInfo.DialServer = func(addr *ServerAddr) (net.Conn, error) {
-		return tls.Dial("tcp", addr.String(), tlsConfig)
-	}
-
-	session, err := DialWithInfo(dialInfo)
+	session, err := Dial(url)
```

Comment:
Could we keep the existing example as it shows how to use a custom TLS config if you're using a private root, etc (but renamed, maybe `ExampleDial_tlsConfig` or the likes?) but add your (much simpler!) example too?

Reply:
I admit I wasn't sure how to update this example. Ok for renaming, that's a good idea. And I will add a new example for this PR.

Reply:
done 4e03a556